src/Main.gren

```diff
@@ -270,13 +270,15 @@ update msg model =
                     Terminal.Run.run 
                         { cpPermission = model.cpPermission
                         , fsPermission = model.fsPermission
+                        , pathToString = model.pathToString
                         , path = path
                         , onInit = RunStarted
                         , onExit = RunExited
                         }
                         |> Task.onError
                             (\error ->
                                 -- TODO: Terminal.Run.errorToString
+                                -- TODO: Why am I not seeing the write to stderr?
```

Comment:
Interesting. Have you tried `Stream.writeStringAsBytes` to see what error pops up?

Also, could it be that `stderr` by default isn't printed to the terminal or something? 🤔

Reply:
I already mentioned this on Discord, but just to have everything in one place:

It seems that when the `update` function receives a `RedirectTerminalIO` message where the `source` stream is closed, it will also close the `target` stream.

That means that once the compiler backend closes, the stdio streams will close for the frontend as well.

I'll fix that on the `main` branch shortly.